esp/templates/program/modules/studentregcore/mainpage.html

```diff
@@ -47,7 +47,7 @@ <h1>Student registration for {{program.niceName}} </h1>
 {% render_inline_program_qsd program "learn:studentregheader" %}
 <div class="alert alert-info">
 <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
-<b>Parents:</b> You should not be at this page!  Please let your kids fill out the registration forms.
+<font color ="red"><big><b>Parents:</b> You should not be at this page!  Please let your kids fill out the registration forms.</big></font>
```

Comment:
Giant and red seems a bit too flashy to me?  I think it's ok as-is -- it's already in an infobox div.

Reply:
I want to make it a little more visible than it currently is. I've gone to a few programs in the past two months, and there are still a lot of parents who seem genuinely surprised to learn that they're not supposed to sign their kids up. I agree that big and red might be a little over the top, though. Maybe big and yellow/orange as a middle ground?

Reply:
You can just change the "alert-info" above to a different bootstrap class.  (And probably also the info-sign.)  cf. http://getbootstrap.com/2.3.2/components.html#alerts